app/src/main/res/layout/dashboard_funding_view.xml

```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  android:id="@+id/creator_dashboard"
+  android:layout_height="match_parent"
+  android:layout_width="match_parent"
+  android:orientation="vertical"
+  xmlns:tools="http://schemas.android.com/tools">
+
+  <TextView
+    android:id="@+id/creator_dashboard_project_name"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:paddingTop="@dimen/grid_new_2"
+    android:paddingLeft="@dimen/grid_new_2"
+    style="@style/Title3"
+    tools:text="ayyyy it's shrek" />
+
+  <TextView
+    android:id="@+id/creator_dashboard_amount_raised"
+    style="@style/Title1"
+    android:layout_marginTop="@dimen/grid_new_3"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:textColor="@color/blue"
+    android:gravity="center_horizontal"
+    tools:text="$12,000" />
+
+  <TextView
+    android:id="@+id/creator_dashboard_funding_text"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:gravity="center_horizontal"
+    android:layout_marginBottom="@dimen/grid_new_3"
+    style="@style/BodyPrimary"
+    tools:text="pledged of $12,000"/>
+
+  <LinearLayout
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="horizontal"
+    android:paddingLeft="@dimen/grid_new_5"
```

Comment:
in generale we're moving away from `Left` and `Right` to use `Start` and `End` instead since L and R are now deprecated